### DISCUSSION TITLE
No formula for mariadb 10.6?

### DISCUSSION THREAD

**Discussion by baffles** [Homebrew/discussions#3130] [Tap maintenance and Homebrew development]
https://github.com/Homebrew/homebrew-core/pull/95021 is the last PR I could find related to MariaDB 10.6. It seems like a `mariadb@10.6` formula should have been added with https://github.com/Homebrew/homebrew-core/pull/95095 ?

Is this a known issue? Version 10.7 isn't LTS and is too bleeding edge. I need to be running version 10.6, but no easy way to install that using homebrew anymore (unless I'm missing something).

**Comment by SMillerDev** [comment]
You can make a pull request for it.

**Reply by baffles** [reply]
Unfortunately, I don't have the time right now to figure out how to build mariadb and create a formula for it. Presumably this would be quite simple for the folks who are already familiar with/maintaining the mariadb formulae?
